subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/CacheLayout.java

```diff
@@ -74,6 +74,7 @@ public enum CacheLayout {
         .changedTo(99, "7.5-rc-1")
         .changedTo(100, "8.0-milestone-5")
         .changedTo(105, "8.1-rc-2")
+        .changedTo(106, "8.2-rc-1")
```

Comment:
📝 This is required, because we update the backing fields of BuildIdentifier which is serialized into the dependency metadata cache.